docker-compose.dev.yml

```diff
@@ -0,0 +1,66 @@
+version: '3'
+
+services:
+  web:
+    build:
+      context: .
+    container_name: kcc3_web
+    volumes:
+      - static_volume:/app/static
+      - media_volume:/app/media
+    depends_on:
+      - db
+      - beat
+    env_file:
+      - ./.env.dev
```

Comment:
nit: these could be specified once using https://docs.docker.com/compose/compose-file/compose-file-v3/#extension-fields. Not a big issue though

Reply:
Done, PTAL now.